models/sariev2024maritime/metadata.yaml

```diff
@@ -0,0 +1,30 @@
+title: Ontology for Digital Twin Development in Maritime Logistics
+acronym: 
+issued: 2024
+modified: 
+contributor:
+ - https://www.linkedin.com/in/bekbolot-sariev-7bb6b0265/
+keyword:
+ - maritime logistics
+ - digital twin
+ - port management
+ - vessel traffic management 
+theme: Subclass H – Social Sciences
+editorialNote: 
+ontologyType:
+ - domain
+language: en
+designedForTask:
+ - conceptual clarification
+ - interoperability
+ - decision support system
+context:
+ - research
+source:
+ - http://essay.utwente.nl/101204/1/Sariev_BA_EEMCS.pdf
```

Comment:
You should use the permanent url provided by the UT: https://purl.utwente.nl/essays/101204